### PR TITLE
Explicitly specify the target triple for native clang test builds

### PR DESCRIPTION
**Pull request by dschuff**
This should make it work regardless of what the default target triple of
clang is.

### CODE REVIEW

**Review by sbc100 — APPROVED** [review]
I still think we should address the windows builder problem rather than working around it.

File: tests/clang_native.py [hunk 1]
```diff
@@ -5,22 +5,37 @@
...
+import platform
+import sys
 from tools.shared import PIPE, run_process, CLANG_CC, CLANG_CXX
 from tools.utils import MACOS, WINDOWS, path_from_root
 
 logger = logging.getLogger('clang_native')
 
+def get_native_triple():
+  arch = {
+      'arm64': 'arm64',
```
Comment (otterley) [inline tests/clang_native.py:17]: should be `aarch64`

Comment (dschuff) [reply tests/clang_native.py:17]: Oh, on MacOS it's `arm64` but clang accepts both. So maybe we need both this existing line and the one you suggested.

**Comment by dschuff** [comment]
> I still think we should address the windows builder problem rather than working around it.

I guess I see this (combined with explicitly specifying clang's default triple to wasm32) as addressing the problem? i.e. IMO the problem is not that Windows now disagrees with other platforms about what the default or native target is; the problem is that the choice of default is affecting us?

**Comment by dschuff** [comment]
Going to push this without waiting for the tests again, since the only change was formatting.

**Pull request merged**